rawdb/rawdb.go

```diff
@@ -165,6 +166,14 @@ func (r *RawDB) Insert(key, value []byte) error {
 		} else {
 			// Encoded coordinates.
 			c := make([]byte, 4+4+4)
+
```

Comment:
I have issues with this fake test. We either have a max size or not. This fixes no problems.